go.mod

```diff
@@ -4,6 +4,8 @@ go 1.14
 
 require (
 	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/umoci v0.4.7
```

Comment:
Could we use `github.com/containers/storage/pkg/idtools` instead? The umoci code is perfectly fine but not yet part of the Podman dependencies. Debian and other distribution had to package it.

Reply:
Sure. I wasn't sure if we could use the storage dependency (I wasn't sure how the inter-repo dependencies are handled in this org). Though I will have to check whether it's trivial to use the storage one (at first glance it really wants you to map (uid,gid) pairs rather than dealing with the mappings separately).

Reply:
My first impressions appears to be correct -- in order to use `github.com/containers/storage/pkg/idtools` you need to map both the user and the group simultaneously, but we don't want to do that. I guess we could just map `0` for the other id but this seems less than ideal (especially since).

If you like I can just copy `idtools.go` into psgo so that we don't add another dependency (or I could submit a patch to storage to allow more "raw" mappings -- at which point it probably would also make sense to switch their types to the `runtime-spec` ones).

Reply:
https://github.com/containers/storage/pull/1101 is the storage PR to allow for "raw" mappings.